Misc/NEWS.d/next/Documentation/2019-05-15-14-31-40.bpo-36567.MNbyoI.rst

```diff
@@ -0,0 +1 @@
+Use debian's manpage service website for references.
```

Comment:
NEWS entry isn't needed for trivial PRs. I've set the "skip news" label.

Reply:
Done